ingest/indexer/service/blockprocessor/full_indexer_block_process_strategy.go

```diff
@@ -105,7 +105,7 @@ func (f *fullIndexerBlockProcessStrategy) processPools(ctx sdk.Context) error {
 	pools := blockPools.GetAll()
 
 	// Process pool pairs
-	if err := f.poolPairPublisher.PublishPoolPairs(ctx, pools); err != nil {
+	if err := f.poolPairPublisher.PublishPoolPairs(ctx, pools, nil); err != nil {
```

Comment:
This seems to imply that pools cannot be created on the start-up block which does not seem to be right. Could you please confirm about why passing nil is safe here?

Reply:
Great point! Changes made.